sno/output_util.py

```diff
@@ -61,7 +61,7 @@ class ExtendedJsonLexer(
         ex_json_lexer = ExtendedJsonLexer()
         # The LexerContext stores the state of the lexer after each call to get_tokens_unprocessed
         lexer_context = pygments.lexer.LexerContext("", 0)
-        formatter = pygments.formatters.TerminalFormatter()
+        formatter = pygments.formatters.Terminal256Formatter(style="colorful")
```

Comment:
unrelated? what's this do? feel free to leave it in if it's useful

Reply:
Ooops was just messing with this last night. It isn't useful